Style fixes and cleanup

f-f opened the pull request:
In this PR:

- fixed some syles, now it looks exactly like the old js version
- removed old JS, as we have the same functionality ported 🎉
- added screenshot:

![screenshot](https://user-images.githubusercontent.com/5480361/33166357-a6b52438-d042-11e7-96d1-77d882a2ce4c.png)

f-f committed:
Upgrade bootstrap to latest beta

f-f committed:
Add old index with old js, for historical record

f-f committed:
Add url to Commit AuthorImg

f-f committed:
Switch toUpper -> capitalize for Pipeline status display

f-f committed:
Add spaces near fontAwesome icons

f-f committed:
Make Project Name bold

f-f committed:
Align all text in table to the left

f-f committed:
Add some margin to fontAwesome icons

f-f committed:
Make display of Job status bigger, and add stacked icons

f-f committed:
Add screenshot to README

f-f committed:
Remove old JS file, will be remembered by us and in git history

zudov commented on src/Dashboard/View.purs:
```diff
@@ -99,7 +101,7 @@ formatStatus { id: PipelineId id, status } =
     [ H.text $ "#" <> show id
     , H.br []
-    , H.text $ String.toUpper $ show status
+    , H.text $ capitalize $ show status
```
No need for `capitalize` really, data constructors are uppercased already and that's what `show` uses.

zudov committed:
Drop 'capitalize', it's already uppercased

No need for capitalize really, data constructors are uppercased
already and that's what show uses.
